client/src/App.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+  import { onMount } from 'svelte'
+  import io, { Socket } from 'socket.io-client'
+
+  let users = []
+  let user: { hash: string; username: string } = JSON.parse(
```

Comment:
This type isn't correct since these properties may not exist on this object as it can be `{}`